app/page.tsx

```diff
@@ -1,16 +1,29 @@
 'use client'
 
 import { Button } from '@/components/ui/button'
+import { useAppDispatch, useAppSelector } from '@/store/hooks'
+import { decrement, increment } from '@/store/testsSlice/testsSlice'
 import { useTheme } from 'next-themes'
 
 export default function Home() {
     const { setTheme } = useTheme()
+    const state = useAppSelector((state) => state)
+    const dispatch = useAppDispatch()
 
     return (
-        <main className='bg-primar flex min-h-screen justify-around p-24'>
-            <h1>DogDB Frontend</h1>
-            <Button onClick={() => setTheme('light')}>Light Mode</Button>
-            <Button onClick={() => setTheme('dark')}>Dark Mode</Button>
+        <main className='bg-primar min-h-screen p-24'>
```

Comment:
Can we update to primary instead of primar

Reply:
Done